docpairs.js

```diff
@@ -0,0 +1,201 @@
+var fs = require('fs');
+const {staticDom} = require('./utils');
+
+function productImageDocPairs() {
+  return [
+    [staticDom(fs.readFileSync(__dirname + '/test/product_classification_test_data/amazon/expected-image.html', 'utf-8')),
+      fs.readFileSync(__dirname + '/test/product_classification_test_data/amazon/source.html', 'utf-8')
```

Comment:
This long line and the ones like it each appear several times. It's bad enough that brfs makes us write the whole thing out a bazillion times; let's minimize it by having a separate function return all the source.html files in a list and then zip them together with the expected data for images, prices, titles, etc.